db/db.go

```diff
@@ -64,15 +65,26 @@ func Open(path string, key tink.AEAD) (*DB, error) {
 	return ret, nil
 }
 
+// ID encapsulates a caller identity. It is required by all database
```

Comment:
Sort of a nit, but we use "ID" for a lot of things. Maybe this ought to be Caller?

I don't feel very strongly about it, though.

Reply:
Yeah, ID is very overloaded. Renamed to Caller.